jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -17,11 +16,13 @@ use std::str::FromStr;
 use tokio::prelude::*;
 
 use self::scalars::{
-    BlockCount, ChainLength, EpochNumber, PoolId, PublicKey, Serial, Slot, TimeOffsetSeconds, Value,
+    BlockCount, ChainLength, EpochNumber, IndexCursor, PoolId, PublicKey, Serial, Slot,
+    TimeOffsetSeconds, Value,
 };
 
 use crate::explorer::{ExplorerDB, Settings};
 
+#[derive(Clone)]
 pub struct Block {
     hash: HeaderHash,
 }
```

Comment:
This is meant to have other fields as needed to send information about blocks, it's not just here to have a hash, right?

Reply:
Well, probably. Right now I'm just asking the "db" for the block's body directly in every resolver, as the "db" is not actually a db but an object in memory. My original idea was to have something like:
```rust
body: Option<ExplorerBlock>
```
or maybe something more granular.

Your concern was about the `Clone` or the fact this could only be a newtype?

Reply:
I was wondering if it's only a newtype with a name that means something else, but yeah, if it's supposed to have other fields later on, it's OK.